ngraph/core/include/ngraph/op/util/broadcast_base.hpp

```diff
@@ -50,6 +50,7 @@ namespace ngraph
 
                 bool evaluate(const HostTensorVector& outputs,
                               const HostTensorVector& inputs) const override;
+                using Op::evaluate;
```

Comment:
Why do we need this?

Reply:
It makes the base class function visible. BroadcastBase class has his own `evaluate `with 3 arguments: 
`                bool evaluate(const HostTensorPtr& arg0,
                              const HostTensorPtr& out,
                              const AxisSet& broadcast_axes) const;`
example: https://onlinegdb.com/B1DZVJOdO

Reply:
It looks strange.
Node already has public virtual evaluate method with three arguments, it should be visible in `BroadcastBase` we don't need to have any usings.
I propose to rename protected method evaluate in this class in order to avoid any issues. with virtual table and RTTI.

Reply:
fixed

Reply:
Ok.